src/settings/window_geometry.rs

```diff
@@ -1,16 +1,63 @@
-#[derive(Debug, Clone)]
+use crate::renderer::Renderer;
+use crate::settings::SETTINGS;
+use crate::window::WindowSettings;
+use glutin::dpi::PhysicalSize;
+use serde::{Deserialize, Serialize};
+use std::path::PathBuf;
+
+#[derive(Debug, Clone, Copy, Serialize, Deserialize)]
 pub struct WindowGeometry {
     pub width: u64,
     pub height: u64,
 }
 
+#[cfg(unix)]
```

Comment:
The feature of a persistent neovide settings file is interesting. Couple of questions (none blocking this):

1. Should it be user readable and documented? Maybe this could be a more approachable way/place to specify multigrid/other features which can't be represented in the neovim config?
2. It feels more general purpose than just window size and sort fits somewhere in the settings module. Would it make sense to expose in a more general way for the rest of the components?

Reply:
> Should it be user readable and documented?
> which can't be represented in the neovim config
Ohh, so sorta as a way to set "defaults" for the command line arguments? Depending on the most common usecase, perhaps put this into a separate file. Do we want "defaults" to take priority or "last adjusted settings"?

> It feels more general purpose than just window size
> Would it make sense to expose in a more general way for the rest of the components?
Ohh yeah definitively!  Without more concrete things to save, I am not sure how to get started programming this in general though. An issue that may arise is having access to all nesscary objects at time of exit. Right now, I'm relying on the fact that the `window_wrapper` object is conveniently in scope during the exit.

Reply:
Ok I think it makes sense to just merge this as is. Was just brainstorming a bit